src/get-started/java-binding.md

```diff
@@ -242,6 +242,38 @@ synchronous I/O in this handler, as it is invoked synchronously in the same
 thread that handles transactions. Blocking that thread will delay transaction
 processing on the node.
 
+### Core Schema API
```

Comment:
Can anyone suggest a better place for this section? Or shall we keep it standalone?

Reply:
I think it's ok to keep it here

Reply:
Then see below — we must explain where it can be used